### PR TITLE
Redis logging

### PR DESCRIPTION
**Pull request by domsj**

### CODE REVIEW

File: docker/alba_centos7_jenkins/Dockerfile [hunk 1]
```diff
@@ -96,7 +98,7 @@ RUN cd isa-l-2.14.0 && make install
 
 # install specific arakoon.
 RUN git clone https://github.com/openvstorage/arakoon.git
-RUN cd arakoon && git pull && git checkout tags/1.8.13
+RUN cd arakoon && git pull && git checkout 4d3a8913abfb5b70a2b8e8427c8e9019ddbe2776
```
Comment (toolslive) [inline docker/alba_centos7_jenkins/Dockerfile]: maybe wait a bit: I sense an imminent arakoon release.

File: ocaml/src/cli_common.ml [hunk 2]
```diff
@@ -17,20 +17,80 @@ limitations under the License.
...
+
+  let hostname = Unix.gethostname () in
+  let component = Printf.sprintf "alba/%s" subcomponent in
+  let pid = Unix.getpid () in
+  let seqnum = ref 0 in
+  let logger = Lwt_log.make
+                 ~output:(fun section level lines ->
+                          let seqnum' = !seqnum in
+                          let () = incr seqnum in
+                          let ts = Core.Time.now () in
```
Comment (toolslive) [inline ocaml/src/cli_common.ml]: core as a dependency in myocamlbuild.ml

File: docker/alba_centos7_jenkins/Dockerfile (hunk 1 above)
Comment (domsj) [inline docker/alba_centos7_jenkins/Dockerfile]: ok, updated to arakoon 1.9.0

File: ocaml/src/cli_common.ml (hunk 2 above)
Comment (domsj) [inline ocaml/src/cli_common.ml]: ok, added

**Pull request merged**